### PR TITLE
fixed invisible input fields

### LINKED ISSUE #3245: Data in one input field disappears if clicked another input field
### Describe the bug
Go to patient-finder modal.

### Actual behavior

https://www.loom.com/share/51b760d7de1a4955a8422cab5200be4e

### Client configuration

- Browser: Chrome Version 80.0.3987.149

### PR DESCRIPTION
**Pull request by adad20**
Fixes #3245

### CODE REVIEW

**Comment by adad20** [comment]
@bradymiller I think there is no need for the class `form-control` and it is causing problem.
If I have done something stupid do let me know 😅

**Comment by sjpadgett** [comment]
@adad20 There is no stupid in programming. There's misinformed, why are they doing it that way but mostly, why the heck aren't they doing it my way! :)

**Comment by adad20** [comment]
> @adad20 There is no stupid in programming. There's misinformed, why are they doing it that way but mostly, why the heck aren't they doing it my way! :)

hi @sjpadgett Thanks it made my day. 😊

File: interface/main/calendar/find_patient_popup.php [hunk 1]
```diff
@@ -124,7 +124,7 @@
     echo "pflag=0"; } ?>'>
     <div class="form-row">
     <label for="searchby" class="col-form-label col-form-label-sm col"><?php echo htmlspecialchars(xl('Search by:'), ENT_NOQUOTES); ?></label>
-   <select name='searchby' id='searchby' class="form-control form-control-sm col">
+   <select name='searchby' id='searchby' class="form-control-sm col">
```
Comment (tywrenn) [inline interface/main/calendar/find_patient_popup.php]: This actually should be `form-control form-control-sm col`

File: interface/main/calendar/find_patient_popup.php [hunk 2]
```diff
@@ -141,7 +141,7 @@
                        } ?>><?php echo htmlspecialchars(xl('DOB'), ENT_NOQUOTES); ?></option>
     </select>
     <label for="searchparm" class="col-form-label col-form-label-sm col"><?php echo htmlspecialchars(xl('for:'), ENT_NOQUOTES); ?></label>
-   <input type='text' class="form-control form-control-sm col" id='searchparm' name='searchparm' size='12' value='<?php echo htmlspecialchars($_REQUEST['searchparm'], ENT_QUOTES); ?>' title='<?php echo htmlspecialchars(xl('If name, any part of lastname or lastname,firstname'), ENT_QUOTES); ?>' />
+   <input type='text' class="form-control-sm col" id='searchparm' name='searchparm' size='12' value='<?php echo htmlspecialchars($_REQUEST['searchparm'], ENT_QUOTES); ?>' title='<?php echo htmlspecialchars(xl('If name, any part of lastname or lastname,firstname'), ENT_QUOTES); ?>' />
```
Comment (tywrenn) [inline interface/main/calendar/find_patient_popup.php]: This actually should be `form-control form-control-sm col`

File: interface/main/calendar/find_patient_popup.php (hunk 1 above)
Comment (adad20) [reply interface/main/calendar/find_patient_popup.php]: Hi @tywrenn but this class is causing the issue I mentioned.
If this is a mistake I will check and correct soon.

Comment (adad20) [reply interface/main/calendar/find_patient_popup.php]: I agree maybe fault is somewhere else. I will commit soon.

**Pull request closed without merging**